src/commands/TrackerCommand.ts

```diff
@@ -0,0 +1,580 @@
+							.setDescription("Blacklist a game for all users.")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("The game which should get blacklisted globaly")
+									.setRequired(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("whitelistgame")
+							.setDescription("Remove a game from the global blacklist")
+							.addStringOption((opt) =>
+								opt
+									.setName("game")
+									.setDescription("The game which should get removed from the global blacklist")
+									.setRequired(true)
+									.setAutocomplete(true)
+							)
+					)
+					.addSubcommand((sub) =>
+						sub
+							.setName("look")
+							.setDescription("take a look into the blacklist of someone else")
+							.addUserOption((opt) =>
+								opt
+									.setName("user")
+									.setDescription("the user of whos blacklist should get shown")
+									.setRequired(true)
+							)
+					)
+			);
+	}
+}
+export default new TrackerCommand();
+
+// commands
+
+async function blacklistAdd(interaction: ChatInputCommandInteraction): Promise<void> {
+	const game = interaction.options.getString("game", false);
+	if (game == null) {
+		activityTrackerBlacklistDb.push("general-user", interaction.user.id);
+		let embed = new EmbedBuilder()
+			.setTitle("Your game activity wont get logged anymore")
+			.setDescription(
+				"Tracking is now disabled for you. To activate it again use `/tracking blacklist remove`"
+			);
+		embed = addDefaultEmbedFooter(embed);
+		await interaction.reply({ embeds: [embed], ephemeral: true });
+		return;
+	}
+
+	if (!activityTrackerBlacklistDb.has(interaction.user.id)) {
+		activityTrackerBlacklistDb.set(interaction.user.id, []);
+	}
+	activityTrackerBlacklistDb.push(interaction.user.id, game);
+
+	let embed = new EmbedBuilder().setTitle(`"${game}" is now on your blacklist`);
+
+	embed = addDefaultEmbedFooter(embed);
+	await interaction.reply({ embeds: [embed], ephemeral: true });
+}
+
+async function blacklistRemove(interaction: ChatInputCommandInteraction): Promise<void> {
+	const game = interaction.options.getString("game", true);
+
+	if (game === "Please activate my tracking again") {
+		const blacklistedUser = activityTrackerBlacklistDb.get("general-user");
+		if (blacklistedUser === undefined) {
+			let embed = new EmbedBuilder().setTitle("Something went wrong");
+			embed = addDefaultEmbedFooter(embed);
+			await interaction.reply({ embeds: [embed], ephemeral: true });
+			return;
+		}
+		blacklistedUser?.filter((e) => e != interaction.user.id);
+		activityTrackerBlacklistDb.set("general-user", blacklistedUser);
+
+		let embed = new EmbedBuilder().setTitle("Your tracking is activated again!");
+		embed = addDefaultEmbedFooter(embed);
+		interaction.reply({ embeds: [embed], ephemeral: true });
+		return;
+	}
+
+	let blacklistedGames: string[] | undefined = activityTrackerBlacklistDb.get(interaction.user.id);
+
+	if (blacklistedGames === undefined) {
```

Comment:
could just put `!` after the get as we know it will be defined

Reply:
I want sure, anymore so I went the save route and did the check manually